lib/rake/task.rb

```diff
@@ -141,7 +141,8 @@ def arg_names
     # Reenable the task, allowing its tasks to be executed if the task
     # is invoked again.
     def reenable
-      @already_invoked = false
+      @already_invoked      = false
```

Comment:
I’m not super opinionated about this change, but this would slightly mess up `git blame` and sometimes it may be more annoying than just looking nice.